cmd/bridge-auth-config-update.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/0chain/gosdk/zcncore"
+	"github.com/spf13/cobra"
+	"log"
+	"strconv"
+	"strings"
+	"sync"
+)
+
+var updateAuthorizerConfigCmd = &cobra.Command{
+	Use:   "bridge-auth-config-update",
+	Short: "Update ZCNSC authorizer settings by ID",
+	Long:  `Update ZCNSC authorizer settings by ID.`,
+	Args:  cobra.MinimumNArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+		const (
+			IDFlag  = "id"
+			FeeFlag = "fee"
+		)
+
+		var (
+			flags = cmd.Flags()
+			ID    string
+			Fee   string
+			err   error
+		)
+
+		if flags.Changed(IDFlag) {
```

Comment:
Return error if "id" is not provided

Reply:
Flag is marked as required, so
it's not necessary
```go
cmd.PersistentFlags().String("id", "", "authorizer ID")
cmd.MarkFlagRequired("id")
```